frontend/src/components/CreatePost.jsx

```
@@ -39,62 +39,70 @@ const CreatePost = ({ userProfile }) => {
+      new URLSearchParams({ content }),
+      {
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded",
+          Authorization: `Bearer ${token}`, // Add token if required by backend
+        },
       }
+    );
+  };
+
+  const createOtherPost = async (token) => {
+    let formData = new FormData();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'formData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)